styles/native/ts/core/widgets/linechart.ts

```diff
@@ -0,0 +1,92 @@
+import { brand, font } from "../variables";
+import { LineChartType } from "../../types/widgets";
+
+/*
+
+DISCLAIMER:
+Do not change this file because it is core styling.
+Customizing core files will make updating Atlas much more difficult in the future.
+To customize any core styling, copy the part you want to customize to styles/native/app/ so the core styling is overwritten.
+
+==========================================================================
+    Line Chart
+
+    Default Class For Mendix Line Chart Widget
+========================================================================== */
+// eslint-disable-next-line @typescript-eslint/camelcase
+export const com_mendix_widget_native_linechart_LineChart: LineChartType = {
+    container: {
+        // All ViewStyle properties are allowed
+    },
+    errorMessage: {
+        // All TextStyle properties are allowed
+        fontFamily: font.family,
+        fontSize: font.sizeSmall,
+    },
+    chart: {
```

Comment:
I don't think we need this prop as we already have `container`

Reply:
`container` view can contain `errorMessage` text or `chart` view. Not sure if combining them is ideal...